src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+  const rentEveryDay = 40;
+  const threeDays = 3;
```

Comment:
```suggestion
  const shortTerm = 3;
```
Не варто в назву змінної включати числове значення. А якщо це значення зміниться? Якщо, наприклад, це вже буде 4 а не 3? Потрібно використовувати назви зі змістом

Reply:
Понял, буду избегать этого. Спасибо!